src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (element array; required)
+`children` specifies the list of elements that will be displayed within the Option component.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the Option component. The default is an empty array. (do I mention the existing classnames here?)
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered Option component. The default is div. Example alternative include li, Button, etc.
+
+### `onOptionSelect` (function, optional)
+`onOptionSelect` is a function that is called when an Option component is selected, by mouse or by keyboard, to perform some additional action.
+
+### `other`
+`other` specifies any additional attributes on the Option component, which can include aria-labelledby, id, key, etc.
+
+## props that this component needs but should only come from the parent
+
+### `isSelected` (boolean; required)
+    `isSelected` specifies whether the option is selected and toggles whether the option is visually selected. The default is false. It should only be passed in as a prop from the ListBox.
```

Comment:
Nit: this text is indented but other lines on the same level aren't.

Reply:
This makes me think that this component should be named `ListBoxOption` like you suggested, since the functionality is tied so closely to `ListBox`.